client/lib/directly/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file Interface to the third-party Real Time Messaging (RTM) widget from Directly.
+ *
+ * @see ./README.md for a higher-level overview
+ * @see https://cloudup.com/cySVQ9R_O6S for Directly's configuration guide
+ */
+
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { loadScript } from 'lib/load-script';
+
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	id: config( 'directly_rtm_widget_id' ),
+	displayAskQuestion: false
+};
+const DIRECTLY_RTM_SCRIPT_URL = 'https://widgets.wp.com/directly/embed.js';
+const DIRECTLY_ASSETS_BASE_URL = 'https://www.directly.com';
+let directlyPromise;
+
+/**
+ * Set up global variables and configuration for the RTM widget
+ *
+ * @see https://cloudup.com/cySVQ9R_O6S for the standard setup instructions
+ */
+function configure() {
+	// Set up the global DirectlyRTM function, required for the RTM widget. This snippet
+	// is pasted from Directly's setup code.
```

Comment:
purely style and not a blocker, but could we move the second sentence fully to a new line? it's not only dangling "This snippet" up there but it's also starting a new idea. I think two lines would be :100: